src/java.base/share/classes/java/lang/Float16.java

```diff
@@ -444,7 +444,7 @@ public static Float16 valueOf(BigDecimal v) {
      * @see Double#isNaN(double)
      */
     public static boolean isNaN(Float16 f16) {
```

Comment:
With changing the implementation if isNaN, isFinite, etc. to bit-based, the regression tests should be updated to cover more cases. In particular, different NaN bit patterns should be tested.

Reply:
Hi @jddarcy , thank you for the review. I will add more input patterns to be tested.

Reply:
Thanks for the update; looks fine.